packages/miniflare/README.md

```diff
@@ -315,6 +315,114 @@ parameter in module format Workers.
     handler. This allows you to access data and functions defined in Node.js
     from your Worker.
 
+<!--prettier-ignore-start-->
+
+- `wrappedBindings?: Record<string, string | { scriptName: string, entrypoint?: string, bindings?: Record<string, Json> }>`
```

Comment:
Unless `scriptName` is to match the capnp implementation, would something like `workerName` fit better here, as we try and move away from the `script` language?

Reply:
That's fair—better to match the API then 👍